plugin.rb

```diff
@@ -263,6 +263,7 @@ def self.onebox_template
       args[:avatar_url] = message.user.avatar_template_url.gsub('{size}', '20')
       args[:cooked] = message.cooked
       args[:created_at] = message.created_at
+      args[:created_at_str] = message.created_at.iso8601
```

Comment:
This solution seems unexpected, can you elaborate please?

We show dates in safari at other places without doing this, no?

Reply:
The message I was seeing in console was:

```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
```

The previous date format was `2022-05-27 18:21:45 UTC`. The right kind of date looks like `2022-05-27T18:21:45Z`.

We do not normally have to do this because Rails serializers do this work. We also call iso8601 some other places too, where we serialize the data ourselves.